docs/docs/api/Dispatcher.md

```diff
@@ -1104,7 +1104,7 @@ The `cache` interceptor implements client-side response caching as described in
 - `store` - The [`CacheStore`](/docs/docs/api/CacheStore.md) to store and retrieve responses from. Default is [`MemoryCacheStore`](/docs/docs/api/CacheStore.md#memorycachestore).
 - `methods` - The [**safe** HTTP methods](https://www.rfc-editor.org/rfc/rfc9110#section-9.2.1) to cache the response of.
 - `cacheByDefault` - The default expiration time to cache responses by if they don't have an explicit expiration. If this isn't present, responses without explicit expiration will not be cached. Default `undefined`.
-- `type` - The type of cache for Undici to act as. Can be `shared` or `private`. Default `shared`.
+- `type` - The type of cache for Undici to act as. Can be `shared` or `private`. Default `shared`. `private` means privately cacheable responses will be cached and potentially shared with other users of your application.
```

Comment:
It would be worth it to reference this to the spec or an article that describes the meaning of it

Reply:
The spec is more permanent than an article which is more likely to turn into a 404, so I would add a link to the RFC. Or MDN maybe, if I find a suitable page on it.

(I assume that you (the Node.js main contributors) wish the documentation to be concise and straight to the point mainly about Node.js itself, so, it should avoid explaining/documenting related concepts that are not Node.js specific. That is why I suggested an edit as concise as I could came up with.)